types/dom-to-image/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for dom-to-image 2.6
+// Project: https://github.com/tsayen/dom-to-image
+// Definitions by: Jip Sterk <https://github.com/JipSterk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="node" />
+
+export namespace DomToImage {
+  interface Options {
+    filter?: (node: Node) => boolean;
+    bgcolor?: string;
+    width?: number;
+    height?: number;
+    style?: {};
+    quality?: number;
+    imagePlaceholder?: string;
+    cachebust?: boolean;
+  }
+}
+
```

Comment:
You want to write this using guidelines specified in https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module
Also update the tests accordingly.